packages/stream-api/src/streamApi.ts

```diff
@@ -1,13 +1,18 @@
 import { WebSocketClient } from './webSocketClient';
 import { UrlProvider } from './url';
 import { parseResponse } from './responses';
-import { vertexvis } from '@vertexvis/frame-streaming-protos';
-import { Disposable, EventDispatcher } from '@vertexvis/utils';
+import { vertexvis, google } from '@vertexvis/frame-streaming-protos';
+import { Disposable, EventDispatcher, UUID } from '@vertexvis/utils';
 
 type ResponseHandler = (
   response: vertexvis.protobuf.stream.IStreamResponse
 ) => void;
 
+interface ParseArgsRequest {
+  requestId?: google.protobuf.IStringValue;
```

Comment:
Do the protobufs require request ID's to be of type `IStringValue`? Or do they support `requestId?: IStringValue | string`? If the later, might be a bit simpler to just represent this as `requestId?: string`.

Reply:
right now, it can be `IStringValue` or `null`.